content/general/concepts/open-source/open-source.md

```diff
@@ -0,0 +1,21 @@
+---
+Title: 'Open Source'
+Description: '"Open source" refers to software that is released with the source code made freely available for modification and possible redistribution.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Conceptual'
+  - 'Development'
+  - 'Linux'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+"Open source" refers to software that is released with the source code made freely available for modification and possible redistribution. Open source projects allow the code to be reused by anyone with the restriction that any derivative projects are released under the same license. Such projects are intended to foster open collaboration, with many developers reviewing and updating the code. One benefit of this is that it provides greater opportunity for bugs and security holes to be detected and fixed. Open source software is also thought to be more trustworthy, since all the code is available to the public, any unwanted or nefarious code will be quickly exposed by the development community surrounding the project.
```

Comment:
```suggestion
The term "open source" refers to softwares that are freely available for modification and possible redistribution. Open source projects allow the source code to be reused by anyone with the restriction that any derivative projects are released under the same license. 

Such projects are intended to foster open collaboration from a variety of perspectives on problem-solving. The code is reviewed and updated by groups of developers of all sizes. One benefit of this is that it provides greater opportunity for bugs and security holes to be detected and handled effectively. 

Open source software (OSS) is also thought to be more trustworthy. Since all the code is available to the public, any unwanted or nefarious behavior or activity will be quickly exposed by the development community surrounding the project.
```